### PR TITLE
fix: receive-applications cronjob utc/pst schedule

### PR DESCRIPTION
**Pull request by marcellmueller**
Implements # \<issue number\>

### CODE REVIEW

File: helm/app/templates/cronJobs/receive-applications.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ metadata:
   labels: {{ include "ccbc.labels" . | nindent 4 }}
 spec:
   suspend: false
-  schedule: "0 9,21 * * *" # run cronjob at At 09:00 and 21:00
+  schedule: "0 1,13 * * *" # run cronjob at At 09:00 and 21:00
```
Comment (VB-at-Bis) [inline helm/app/templates/cronJobs/receive-applications.yaml:8]: I'd expect 5 and 17  :) .. we are PST (which is UTC +8), so 9am PST means 17 UTC (https://www.google.com/search?q=9am+pst+in+utc) and 9pm PST (21) means 5am.

**Review by VB-at-Bis — CHANGES_REQUESTED** [review]
see inline

File: helm/app/templates/cronJobs/receive-applications.yaml (hunk 1 above)
Comment (marcellmueller) [reply helm/app/templates/cronJobs/receive-applications.yaml]: Fixed, thanks Vlad appreciate it :)

**Review by VB-at-Bis — APPROVED** [review]
looks good

**Pull request merged**